.eslintcache

```diff
@@ -0,0 +1 @@
"column":13,"nodeType":"142","messageId":"143","endLine":28,"endColumn":24},{"ruleId":"140","severity":1,"message":"141","line":43,"column":13,"nodeType":"142","messageId":"143","endLine":43,"endColumn":24},{"ruleId":"140","severity":1,"message":"141","line":48,"column":13,"nodeType":"142","messageId":"143","endLine":48,"endColumn":26},{"ruleId":"140","severity":1,"message":"141","line":44,"column":9,"nodeType":"142","messageId":"143","endLine":44,"endColumn":20},{"ruleId":"140","severity":1,"message":"141","line":45,"column":9,"nodeType":"142","messageId":"143","endLine":45,"endColumn":20},{"ruleId":"140","severity":1,"message":"141","line":46,"column":9,"nodeType":"142","messageId":"143","endLine":46,"endColumn":20},{"ruleId":"140","severity":1,"message":"141","line":74,"column":13,"nodeType":"142","messageId":"143","endLine":74,"endColumn":24},{"ruleId":"140","severity":1,"message":"141","line":118,"column":13,"nodeType":"142","messageId":"143","endLine":118,"endColumn":24},{"ruleId":"140","severity":1,"message":"141","line":135,"column":13,"nodeType":"142","messageId":"143","endLine":135,"endColumn":24},{"ruleId":"140","severity":1,"message":"141","line":181,"column":13,"nodeType":"142","messageId":"143","endLine":181,"endColumn":24},{"ruleId":"140","severity":1,"message":"141","line":202,"column":13,"nodeType":"142","messageId":"143","endLine":202,"endColumn":24},{"ruleId":"140","severity":1,"message":"141","line":37,"column":46,"nodeType":"142","messageId":"143","endLine":37,"endColumn":59},{"ruleId":"140","severity":1,"message":"141","line":39,"column":43,"nodeType":"142","messageId":"143","endLine":39,"endColumn":56},{"ruleId":"144","severity":1,"message":"145","line":47,"column":103,"nodeType":"146","messageId":"147","endLine":47,"endColumn":104},{"ruleId":"140","severity":1,"message":"141","line":48,"column":13,"nodeType":"142","messageId":"143","endLine":48,"endColumn":24},{"ruleId":"140","severity":1,"message":"141","line":94,"column":96,"nodeType":"142","messageId":"143","endLine":94,"endColumn":109},{"ruleId":"144","severity":1,"message":"145","line":108,"column":104,"nodeType":"146","messageId":"147","endLine":108,"endColumn":105},{"ruleId":"140","severity":1,"message":"141","line":120,"column":23,"nodeType":"142","messageId":"143","endLine":120,"endColumn":36},{"ruleId":"140","severity":1,"message":"141","line":96,"column":17,"nodeType":"142","messageId":"143","endLine":96,"endColumn":28},{"ruleId":"140","severity":1,"message":"141","line":111,"column":17,"nodeType":"142","messageId":"143","endLine":111,"endColumn":30},"Parsing error: \"parserOptions.project\" has been set for @typescript-eslint/parser.\nThe file does not match your project config: mod.ts.\nThe file must be included in at least one of the projects provided.","no-console","Unexpected console statement.","MemberExpression","unexpected","@typescript-eslint/no-unused-vars","'e' is defined but never used. Allowed unused args must match /^_/u.","Identifier","unusedVar"]
```

Comment:
did you mean to include this file? This looks like something that shouldn't be in version control

Reply:
Nope. thanks for catching that.